### ISSUE TITLE
Mapping between v1.05 and v2.01

### ISSUE THREAD

**Issue by tvldev02** [BD-IATI/edi#1]
Following tags are mismatched.

# Location tag
## In v1.05
```
<location>
		  <name>Dhaka</name>
		  <description>Dhaka, Bangladesh</description>
		  <point srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
			<pos>23.7099210000000010 90.4071430000000050</pos>
		  </point>
		  <exactness code="1"/>
		  <location-reach code="1"/>
		  <location-class code="2"/>
		  <feature-designation code="PPLC"/>
</location>
```
## In v2.01
 ```
<location>
          <location-reach code="1"/>
          <location-id vocabulary="G1" code="1185241"/> 		//v1.05 does not contain
          <administrative vocabulary="G1" level="1" code="1185241"/> 	//v1.05 does not contain
          <point srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
            <pos>23.7104 90.40744</pos>
          </point>
          <exactness code="2"/>
          <location-class code="1"/>
          <feature-designation code="PPLC"/>
</location>
```

# planned-disbursement tag
## In v1.05
`	not exists`
## In v2.01
```
<planned-disbursement>
        <period-start iso-date="2015-04-01"/>
        <period-end iso-date="2016-03-31"/>
        <value value-date="2007-10-03">100000</value>
</planned-disbursement>
```

# transaction tag
## In v1.05
```
<transaction>
        <value currency="GBP" value-date="2008-12-31">7088</value>
        <transaction-type code="E">Expenditure</transaction-type>
        <provider-org ref="GB-1">Department for International Development</provider-org>
        <description>Aggregated spend data - Financial Year 2008 Quarter 3</description>
        <transaction-date iso-date="2008-12-31"/>
</transaction>
```
## In v2.01
```
<transaction>
        <transaction-type code="2"/>
        <transaction-date iso-date="2007-10-03"/>
        <value value-date="2007-10-03">500000.00</value>
        <provider-org provider-activity-id="CA-3"/> 	//provider-activity-id = ? field in v1.05 
        <flow-type code="10"/> 				//v1.05 does not contain
        <tied-status code="5"/> 			//v1.05 does not contain
</transaction>
```

# conditions tag
## In v1.05
`	not exists`
## In v2.01
```
<conditions attached="1">
        <condition type="1">
          <narrative xml:lang="en">1.1, 1.2, 1.3,  www.international.gc.ca/iati-conditions</narrative>
          <narrative xml:lang="fr">1.1, 1.2, 1.3,  www.international.gc.ca/iita-conditions</narrative>
        </condition>
        <condition type="3">
          <narrative xml:lang="en">3.1,  www.international.gc.ca/iati-conditions</narrative>
          <narrative xml:lang="fr">3.1,  www.international.gc.ca/iita-conditions</narrative>
        </condition>
</conditions>
```

**Comment by markbrough** [comment]
1.05 *can* contain these elements, but they are not required. The data will look different from different publishers - it is generally a bad idea to expect certain tags to exist unless we really require them.

1. Location tag:
http://iatistandard.org/105/activity-standard/iati-activities/iati-activity/location/
2. Transactions tag:
http://iatistandard.org/105/activity-standard/iati-activities/iati-activity/transaction/
3. Conditions tag:
http://iatistandard.org/105/activity-standard/iati-activities/iati-activity/conditions/

**Comment by markbrough** [comment]
Assuming you're working on the [conversion](http://bd-iati.github.io/development/sprint-1/#conversion) component, I would try focus first on the few elements identified in there as important to be converted, rather than trying to do everything.

**Comment by tvldev02** [comment]
Sample result of our conversion component.

The downloaded v1.05 xml (Only two activity)

```
<result xmlns:iati-extra="http://datastore.iatistandard.org/ns">
  <ok>True</ok>
  <iati-activities generated-datetime='2016-02-15T12:57:03.923112'>
    <query>
      <total-count>379</total-count>
      <start></start>
      <limit></limit>
    </query>
    <iati-activity iati-extra:version="1.05" last-updated-datetime="2014-05-05T05:35:09" xml:lang="en" default-currency="GBP" hierarchy="2">
      <reporting-org ref="GB-1" type="10">Department for International Development</reporting-org>
      <participating-org ref="GB" type="10" role="Funding">UNITED KINGDOM</participating-org>
      <participating-org ref="GB-1" type="10" role="Extending">Department for International Development</participating-org>
      <participating-org ref="22000" role="Implementing">Donor country-based NGO</participating-org>
      <recipient-country code="BD">Bangladesh</recipient-country>
      <collaboration-type code="3">Bilateral, core contributions to NGOs and other private bodies / PPPs</collaboration-type>
      <default-flow-type code="10">ODA</default-flow-type>
      <default-aid-type code="C01">Project-type interventions</default-aid-type>
      <default-finance-type code="110">Aid grant excluding debt reorganisation</default-finance-type>
      <iati-identifier>GB-1-114209-101</iati-identifier>
      <other-identifier owner-ref="GB-1" owner-name="DFID">114209-101</other-identifier>
      <title>NOTFORPROFITORG and P0118 for Education for Children with Disabilities (CWD) CSCF443</title>
      <description>Title: NOTFORPROFITORG and P0118 for Education for Children with Disabilities (CWD) CSCF443</description>
      <sector code="15150" percentage="55">Democratic participation and civil society</sector>
      <sector code="15160" percentage="35">Human rights</sector>
      <sector code="11220" percentage="10">Primary education</sector>
      <activity-date type="end-actual" iso-date="2012-03-31">2012-03-31</activity-date>
      <activity-date type="end-planned" iso-date="2012-03-31">2012-03-31</activity-date>
      <activity-date type="start-actual" iso-date="2008-07-01">2008-07-01</activity-date>
      <activity-date type="start-planned" iso-date="2008-07-01">2008-07-01</activity-date>
      <activity-status code="4">Post-completion</activity-status>
      <contact-info>
        <organisation>Department for International Development</organisation>
        <telephone>+44 (0) 1355 84 3132</telephone>
        <email>enquiry@dfid.gov.uk</email>
        <mailing-address>Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</mailing-address>
      </contact-info>
      <default-tied-status code="5">Untied</default-tied-status>
      <policy-marker code="1" vocabulary="DAC" significance="0">Gender Equality</policy-marker>
      <policy-marker code="2" vocabulary="DAC" significance="0">Aid to Environment</policy-marker>
      <policy-marker code="3" vocabulary="DAC" significance="2">Participatory Development/Good Governance</policy-marker>
      <policy-marker code="4" vocabulary="DAC" significance="0">Trade Development</policy-marker>
      <policy-marker code="5" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Convention on Biological Diversity</policy-marker>
      <policy-marker code="6" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Framework Convention on Climate Change - Mitigation</policy-marker>
      <policy-marker code="7" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Framework Convention on Climate Change - Adaptation</policy-marker>
      <policy-marker code="8" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Convention to Combat Desertification</policy-marker>
      <transaction>
        <value value-date="2008-07-01">44173</value>
        <transaction-type code="C">Commitment</transaction-type>
        <transaction-date iso-date="2008-07-01">Total Commitment to 31 Mar 2012</transaction-date>
      </transaction>
      <transaction>
        <value currency="GBP" value-date="2008-12-31">7088</value>
        <transaction-type code="E">Expenditure</transaction-type>
        <provider-org ref="GB-1">Department for International Development</provider-org>
        <description>Aggregated spend data - Financial Year 2008 Quarter 3</description>
        <transaction-date iso-date="2008-12-31"/>
      </transaction>
      <transaction>
        <value currency="GBP" value-date="2009-03-31">37085</value>
        <transaction-type code="E">Expenditure</transaction-type>
        <provider-org ref="GB-1">Department for International Development</provider-org>
        <description>Aggregated spend data - Financial Year 2008 Quarter 4</description>
        <transaction-date iso-date="2009-03-31"/>
      </transaction>
      <budget type="Original">
        <period-start iso-date="2008-04-01"/>
        <period-end iso-date="2008-06-30"/>
        <value value-date="2008-07-01" currency="GBP">44173</value>
      </budget>
      <related-activity ref="GB-1-114209" type="1">Education for Children with Disabilities (CWD) CSCF443</related-activity>
      <related-activity ref="GB-1-114209-102" type="3">Accountable Grant CSCF0443: PROMOTING RIGHTS AND ACCESS TO EDUCATION FOR CHILDREN WITH DISABILITIES</related-activity>
      <capital-spend percentage="0"/>
      <country-budget-items vocabulary="1">
        <budget-item code="1.7.1" percentage="55">Other General Services - General Public Service - support to civil society </budget-item>
        <budget-item code="2.1.6" percentage="35">Justice, Law and Order - Justice, Law, Order and Security - human rights affairs</budget-item>
        <budget-item code="5.3.3" percentage="10">Education - Social Affairs - pre-primary</budget-item>
      </country-budget-items>
      <legacy-data name="11220 Primary Education (MDG Education)" value="11220" iati-equivalent="sector">10%</legacy-data>
      <legacy-data name="15150 Strengthening Civil Society: (Governance and Security)" value="15150" iati-equivalent="sector">55%</legacy-data>
      <legacy-data name="15162 Human Rights: (Governance and Security)" value="15162" iati-equivalent="sector">35%</legacy-data>
    </iati-activity>
    <iati-activity iati-extra:version="1.05" last-updated-datetime="2015-12-01T03:03:07" xml:lang="en" default-currency="GBP" hierarchy="2">
      <reporting-org ref="GB-1" type="10">Department for International Development</reporting-org>
      <participating-org ref="GB" type="10" role="Funding">UNITED KINGDOM</participating-org>
      <participating-org ref="GB-1" type="10" role="Extending">Department for International Development</participating-org>
      <participating-org ref="11000" role="Implementing">Donor Government</participating-org>
      <recipient-country code="BD">Bangladesh</recipient-country>
      <collaboration-type code="1">Bilateral</collaboration-type>
      <default-flow-type code="10">ODA</default-flow-type>
      <default-aid-type code="C01">Project-type interventions</default-aid-type>
      <default-finance-type code="110">Aid grant excluding debt reorganisation</default-finance-type>
      <iati-identifier>GB-1-202976-108</iati-identifier>
      <other-identifier owner-ref="GB-1" owner-name="DFID">202976-108</other-identifier>
      <title>Strengthening of SREDA and Dissemination of Solar Lantern in Bangladesh</title>
      <description>Title: Strengthening of SREDA and Dissemination of Solar Lantern in Bangladesh</description>
      <sector code="41010">Environmental policy and administrative management</sector>
      <activity-date type="end-planned" iso-date="2017-03-31">2017-03-31</activity-date>
      <activity-date type="start-actual" iso-date="2015-01-01">2015-01-01</activity-date>
      <activity-date type="start-planned" iso-date="2015-01-01">2015-01-01</activity-date>
      <activity-status code="2">Implementation</activity-status>
      <contact-info>
        <organisation>Department for International Development</organisation>
        <telephone>+44 (0) 1355 84 3132</telephone>
        <email>enquiry@dfid.gov.uk</email>
        <mailing-address>Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</mailing-address>
      </contact-info>
      <default-tied-status code="5">Untied</default-tied-status>
      <policy-marker code="1" vocabulary="DAC" significance="1">Gender Equality</policy-marker>
      <policy-marker code="2" vocabulary="DAC" significance="2">Aid to Environment</policy-marker>
      <policy-marker code="3" vocabulary="DAC" significance="0">Participatory Development/Good Governance</policy-marker>
      <policy-marker code="4" vocabulary="DAC" significance="0">Trade Development</policy-marker>
      <policy-marker code="5" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Convention on Biological Diversity</policy-marker>
      <policy-marker code="6" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Framework Convention on Climate Change - Mitigation</policy-marker>
      <policy-marker code="7" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Framework Convention on Climate Change - Adaptation</policy-marker>
      <policy-marker code="8" vocabulary="DAC" significance="0">Aid Targeting the Objectives of the Convention to Combat Desertification</policy-marker>
      <transaction>
        <value value-date="2015-01-01">754496</value>
        <transaction-type code="C">Commitment</transaction-type>
        <transaction-date iso-date="2015-01-01"/>
      </transaction>
      <transaction>
        <value currency="GBP" value-date="2015-11-30">90000</value>
        <transaction-type code="D">Disbursement</transaction-type>
        <provider-org ref="GB-1">Department for International Development</provider-org>
        <receiver-org ref="11000">Deutsche Gesellschaft fuer Internationale Zusammenarbeit (GIZ) GmbH</receiver-org>
        <description>Aid to international organisations</description>
        <transaction-date iso-date="2015-11-30"/>
      </transaction>
      <budget type="Revised">
        <period-start iso-date="2015-07-01"/>
        <period-end iso-date="2015-09-30"/>
        <value value-date="2015-01-01" currency="GBP">604496</value>
      </budget>
      <budget type="Revised">
        <period-start iso-date="2016-04-01"/>
        <period-end iso-date="2016-06-30"/>
        <value value-date="2015-01-01" currency="GBP">150000</value>
      </budget>
      <related-activity ref="GB-1-202976" type="1">Providing Clean Energy to the Rural Poor of Bangladesh</related-activity>
      <related-activity ref="GB-1-202976-101" type="3">Programme cost to support Infrastructure Development Company Limited for implementing solar home system and piloting mini grid system</related-activity>
      <related-activity ref="GB-1-202976-102" type="3">Programme capital cost to support Infrastructure Development Company Limited for implementing solar home system and piloting mini grid system</related-activity>
      <related-activity ref="GB-1-202976-106" type="3">Programme cost to support Infrastructure Development Company Limited for implementing solar home system and piloting mini grid system- ICF RDEL</related-activity>
      <related-activity ref="GB-1-202976-107" type="3">Programme capital cost to support Infrastructure Development Company Limited for implementing solar home system and piloting mini grid system- ICF CDEL</related-activity>
      <capital-spend percentage="100"/>
      <country-budget-items vocabulary="1">
        <budget-item code="4.2.1" percentage="100">Environment - Water, Natural Resource Management and Environment - policy, planning and administration</budget-item>
      </country-budget-items>
      <legacy-data name="Climate Change, Environment - Low Carbon" value="41090" iati-equivalent="sector"/>
    </iati-activity>

	<iati-activity iati-extra:version="1.05" last-updated-datetime="2016-01-14T02:46:03" xml:lang="en" default-currency="GBP" hierarchy="1">
	<reporting-org ref="GB-1" type="10">Department for International Development</reporting-org><participating-org ref="GB" type="10" role="Funding">UNITED KINGDOM</participating-org><participating-org ref="GB-1" type="10" role="Extending">Department for International Development</participating-org><participating-org ref="20000" role="Implementing">NON-GOVERNMENTAL ORGANISATIONS (NGOs) AND CIVIL SOCIETY</participating-org><participating-org ref="52000" role="Implementing">Other.</participating-org><recipient-country code="BD" percentage="100">Bangladesh</recipient-country><iati-identifier>GB-1-202958</iati-identifier><other-identifier owner-ref="GB-1" owner-name="DFID">202958</other-identifier><title>Creating Opportunities for the Poor and Excluded in Bangladesh (COPE)</title><description>To work with the most poor and marginalised people in Bangladesh to make them aware of their rights, and provide advice, advocacy support and organisation to claim those rights in practice.  As a result of the programme, by December 2016, people who would otherwise be excluded from the benefits of development will gain access to services, assets and economic opportunities, and be given a greater say in decisions affecting their lives.</description><sector code="15150" percentage="50">Democratic participation and civil society</sector><sector code="15160" percentage="30">Human rights</sector><sector code="15111" percentage="20">Public finance management</sector><activity-date type="end-planned" iso-date="2016-12-31">2016-12-31</activity-date><activity-date type="start-actual" iso-date="2013-07-15">2013-07-15</activity-date><activity-date type="start-planned" iso-date="2013-07-01">2013-07-01</activity-date><activity-status code="2">Implementation</activity-status><contact-info><organisation>Department for International Development</organisation><telephone>+44 (0) 1355 84 3132</telephone><email>enquiry@dfid.gov.uk</email><mailing-address>Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</mailing-address></contact-info><default-tied-status code="5">Untied</default-tied-status><related-activity ref="GB-1-202958-101" type="2">Creating Opportunities for the Poor and Excluded in Bangladesh (COPE) Implementation phase managed by Manusher Jonno Foundation</related-activity><related-activity ref="GB-1-202958-102" type="2">Creating Opportunities for the Poor and Excluded (COPE): Technical support for results monitoring</related-activity><related-activity ref="GB-1-202958-103" type="2">Creating Opportunities for the Poor and Excluded (COPE): Independent Evaluation</related-activity><document-link url="http://iati.dfid.gov.uk/iati_documents/3804429.xlsx" format="application/vnd.ms-excel"><title>Logical framework 202958 (July, 2013).xlsx</title><language code="en">English</language><category code="A02">Objectives / Purpose of activity</category><category code="A08">Results, outcomes and outputs</category></document-link><document-link url="http://iati.dfid.gov.uk/iati_documents/4019077.odt" format="application/vnd.oasis.opendocument.text"><title>Business Case and Summary 202958 (January, 2015).odt</title><language code="en">English</language><category code="A05">Budget</category><category code="A04">Conditions</category><category code="A03">Intended ultimate beneficiaries</category><category code="A02">Objectives / Purpose of activity</category><category code="A01">Pre- and post-project impact appraisal</category></document-link><document-link url="http://iati.dfid.gov.uk/iati_documents/4685171.odt" format="application/vnd.oasis.opendocument.text"><title>Annual review (1) 202958 (May, 2015).odt</title><language code="en">English</language><category code="A08">Results, outcomes and outputs</category><category code="A07">Review of project performance and evaluation</category></document-link><document-link url="http://iati.dfid.gov.uk/iati_documents/5062308.odt" format="application/vnd.oasis.opendocument.text"><title>Annual review (2) 202958 (July, 2015).odt</title><language code="en">English</language><category code="A08">Results, outcomes and outputs</category><category code="A07">Review of project performance and evaluation</category></document-link><conditions attached="false"/>
	</iati-activity>
  </iati-activities>
</result>
```

## After converting to v2.01 using our component is


```
<?xml version="1.0" encoding="utf-8"?>
<result xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <iati-activities generated-datetime="2016-02-15T12:57:03.923112">
    <iati-activity last-updated-datetime="2014-05-05T05:35:09" xml:lang="en" default-currency="GBP" hierarchy="2" iati-extra:version="2.01" xmlns:iati-extra="http://datastore.iatistandard.org/ns">
      <iati-identifier>GB-1-114209-101</iati-identifier>
      <reporting-org ref="GB-1" type="10">
        <narrative xml:lang="en">Department for International Development</narrative>
      </reporting-org>
      <title>
        <narrative xml:lang="en">NOTFORPROFITORG and P0118 for Education for Children with Disabilities (CWD) CSCF443</narrative>
      </title>
      <description>
        <narrative xml:lang="en">Title: NOTFORPROFITORG and P0118 for Education for Children with Disabilities (CWD) CSCF443</narrative>
      </description>
      <participating-org ref="GB" type="10" role="1">
        <narrative xml:lang="en">UNITED KINGDOM</narrative>
      </participating-org>
      <participating-org ref="GB-1" type="10" role="3">
        <narrative xml:lang="en">Department for International Development</narrative>
      </participating-org>
      <participating-org ref="22000" role="4">
        <narrative xml:lang="en">Donor country-based NGO</narrative>
      </participating-org>
      <other-identifier owner-ref="GB-1" owner-name="DFID" />
      <activity-status code="4" />
      <activity-date type="4" iso-date="2012-03-31" />
      <activity-date type="3" iso-date="2012-03-31" />
      <activity-date type="2" iso-date="2008-07-01" />
      <activity-date type="1" iso-date="2008-07-01" />
      <contact-info>
        <organisation>
          <narrative xml:lang="en">Department for International Development</narrative>
        </organisation>
        <telephone>+44 (0) 1355 84 3132</telephone>
        <email>enquiry@dfid.gov.uk</email>
        <mailing-address>
          <narrative xml:lang="en">Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</narrative>
        </mailing-address>
      </contact-info>
      <recipient-country code="BD" />
      <sector code="15150" percentage="55" />
      <sector code="15160" percentage="35" />
      <sector code="11220" percentage="10" />
      <country-budget-items vocabulary="1">
        <budget-item code="1.7.1" percentage="55" />
        <budget-item code="2.1.6" percentage="35" />
        <budget-item code="5.3.3" percentage="10" />
      </country-budget-items>
      <policy-marker vocabulary="DAC" code="1" significance="0" />
      <policy-marker vocabulary="DAC" code="2" significance="0" />
      <policy-marker vocabulary="DAC" code="3" significance="2" />
      <policy-marker vocabulary="DAC" code="4" significance="0" />
      <policy-marker vocabulary="DAC" code="5" significance="0" />
      <policy-marker vocabulary="DAC" code="6" significance="0" />
      <policy-marker vocabulary="DAC" code="7" significance="0" />
      <policy-marker vocabulary="DAC" code="8" significance="0" />
      <collaboration-type code="3" />
      <default-flow-type code="10" />
      <default-finance-type code="110" />
      <default-aid-type code="C01" />
      <default-tied-status code="5" />
      <budget type="1">
        <period-start iso-date="2008-04-01" />
        <period-end iso-date="2008-06-30" />
        <value currency="GBP" value-date="2008-07-01">44173</value>
      </budget>
      <capital-spend percentage="0" />
      <transaction>
        <transaction-type code="2" />
        <transaction-date iso-date="2008-07-01" />
        <value value-date="2008-07-01">44173</value>
      </transaction>
      <transaction>
        <transaction-type code="4" />
        <transaction-date iso-date="2008-12-31" />
        <value currency="GBP" value-date="2008-12-31">7088</value>
        <description />
        <provider-org ref="GB-1" />
      </transaction>
      <transaction>
        <transaction-type code="4" />
        <transaction-date iso-date="2009-03-31" />
        <value currency="GBP" value-date="2009-03-31">37085</value>
        <description />
        <provider-org ref="GB-1" />
      </transaction>
      <related-activity ref="GB-1-114209" type="1" />
      <related-activity ref="GB-1-114209-102" type="3" />
      <legacy-data name="11220 Primary Education (MDG Education)" value="11220" iati-equivalent="sector" />
      <legacy-data name="15150 Strengthening Civil Society: (Governance and Security)" value="15150" iati-equivalent="sector" />
      <legacy-data name="15162 Human Rights: (Governance and Security)" value="15162" iati-equivalent="sector" />
    </iati-activity>
    <iati-activity last-updated-datetime="2015-12-01T03:03:07" xml:lang="en" default-currency="GBP" hierarchy="2" iati-extra:version="2.01" xmlns:iati-extra="http://datastore.iatistandard.org/ns">
      <iati-identifier>GB-1-202976-108</iati-identifier>
      <reporting-org ref="GB-1" type="10">
        <narrative xml:lang="en">Department for International Development</narrative>
      </reporting-org>
      <title>
        <narrative xml:lang="en">Strengthening of SREDA and Dissemination of Solar Lantern in Bangladesh</narrative>
      </title>
      <description>
        <narrative xml:lang="en">Title: Strengthening of SREDA and Dissemination of Solar Lantern in Bangladesh</narrative>
      </description>
      <participating-org ref="GB" type="10" role="1">
        <narrative xml:lang="en">UNITED KINGDOM</narrative>
      </participating-org>
      <participating-org ref="GB-1" type="10" role="3">
        <narrative xml:lang="en">Department for International Development</narrative>
      </participating-org>
      <participating-org ref="11000" role="4">
        <narrative xml:lang="en">Donor Government</narrative>
      </participating-org>
      <other-identifier owner-ref="GB-1" owner-name="DFID" />
      <activity-status code="2" />
      <activity-date type="3" iso-date="2017-03-31" />
      <activity-date type="2" iso-date="2015-01-01" />
      <activity-date type="1" iso-date="2015-01-01" />
      <contact-info>
        <organisation>
          <narrative xml:lang="en">Department for International Development</narrative>
        </organisation>
        <telephone>+44 (0) 1355 84 3132</telephone>
        <email>enquiry@dfid.gov.uk</email>
        <mailing-address>
          <narrative xml:lang="en">Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</narrative>
        </mailing-address>
      </contact-info>
      <recipient-country code="BD" />
      <sector code="41010" />
      <country-budget-items vocabulary="1">
        <budget-item code="4.2.1" percentage="100" />
      </country-budget-items>
      <policy-marker vocabulary="DAC" code="1" significance="1" />
      <policy-marker vocabulary="DAC" code="2" significance="2" />
      <policy-marker vocabulary="DAC" code="3" significance="0" />
      <policy-marker vocabulary="DAC" code="4" significance="0" />
      <policy-marker vocabulary="DAC" code="5" significance="0" />
      <policy-marker vocabulary="DAC" code="6" significance="0" />
      <policy-marker vocabulary="DAC" code="7" significance="0" />
      <policy-marker vocabulary="DAC" code="8" significance="0" />
      <collaboration-type code="1" />
      <default-flow-type code="10" />
      <default-finance-type code="110" />
      <default-aid-type code="C01" />
      <default-tied-status code="5" />
      <budget type="2">
        <period-start iso-date="2015-07-01" />
        <period-end iso-date="2015-09-30" />
        <value currency="GBP" value-date="2015-01-01">604496</value>
      </budget>
      <budget type="2">
        <period-start iso-date="2016-04-01" />
        <period-end iso-date="2016-06-30" />
        <value currency="GBP" value-date="2015-01-01">150000</value>
      </budget>
      <capital-spend percentage="100" />
      <transaction>
        <transaction-type code="2" />
        <transaction-date iso-date="2015-01-01" />
        <value value-date="2015-01-01">754496</value>
      </transaction>
      <transaction>
        <transaction-type code="3" />
        <transaction-date iso-date="2015-11-30" />
        <value currency="GBP" value-date="2015-11-30">90000</value>
        <description />
        <provider-org ref="GB-1" />
        <receiver-org ref="11000" />
      </transaction>
      <related-activity ref="GB-1-202976" type="1" />
      <related-activity ref="GB-1-202976-101" type="3" />
      <related-activity ref="GB-1-202976-102" type="3" />
      <related-activity ref="GB-1-202976-106" type="3" />
      <related-activity ref="GB-1-202976-107" type="3" />
      <legacy-data name="Climate Change, Environment - Low Carbon" value="41090" iati-equivalent="sector" />
    </iati-activity>
    <iati-activity last-updated-datetime="2016-01-14T02:46:03" xml:lang="en" default-currency="GBP" hierarchy="1" iati-extra:version="2.01" xmlns:iati-extra="http://datastore.iatistandard.org/ns">
      <iati-identifier>GB-1-202958</iati-identifier>
      <reporting-org ref="GB-1" type="10">
        <narrative xml:lang="en">Department for International Development</narrative>
      </reporting-org>
      <title>
        <narrative xml:lang="en">Creating Opportunities for the Poor and Excluded in Bangladesh (COPE)</narrative>
      </title>
      <description>
        <narrative xml:lang="en">To work with the most poor and marginalised people in Bangladesh to make them aware of their rights, and provide advice, advocacy support and organisation to claim those rights in practice.  As a result of the programme, by December 2016, people who would otherwise be excluded from the benefits of development will gain access to services, assets and economic opportunities, and be given a greater say in decisions affecting their lives.</narrative>
      </description>
      <participating-org ref="GB" type="10" role="1">
        <narrative xml:lang="en">UNITED KINGDOM</narrative>
      </participating-org>
      <participating-org ref="GB-1" type="10" role="3">
        <narrative xml:lang="en">Department for International Development</narrative>
      </participating-org>
      <participating-org ref="20000" role="4">
        <narrative xml:lang="en">NON-GOVERNMENTAL ORGANISATIONS (NGOs) AND CIVIL SOCIETY</narrative>
      </participating-org>
      <participating-org ref="52000" role="4">
        <narrative xml:lang="en">Other.</narrative>
      </participating-org>
      <other-identifier owner-ref="GB-1" owner-name="DFID" />
      <activity-status code="2" />
      <activity-date type="3" iso-date="2016-12-31" />
      <activity-date type="2" iso-date="2013-07-15" />
      <activity-date type="1" iso-date="2013-07-01" />
      <contact-info>
        <organisation>
          <narrative xml:lang="en">Department for International Development</narrative>
        </organisation>
        <telephone>+44 (0) 1355 84 3132</telephone>
        <email>enquiry@dfid.gov.uk</email>
        <mailing-address>
          <narrative xml:lang="en">Public Enquiry Point, Abercrombie House, Eaglesham Road, East Kilbride, Glasgow G75 8EA</narrative>
        </mailing-address>
      </contact-info>
      <recipient-country code="BD" percentage="100" />
      <sector code="15150" percentage="50" />
      <sector code="15160" percentage="30" />
      <sector code="15111" percentage="20" />
      <default-tied-status code="5" />
      <document-link url="http://iati.dfid.gov.uk/iati_documents/3804429.xlsx" format="application/vnd.ms-excel">
        <title>
          <narrative xml:lang="en">Logical framework 202958 (July, 2013).xlsx</narrative>
        </title>
        <category code="A02" />
        <category code="A08" />
        <language code="en" />
      </document-link>
      <document-link url="http://iati.dfid.gov.uk/iati_documents/4019077.odt" format="application/vnd.oasis.opendocument.text">
        <title>
          <narrative xml:lang="en">Business Case and Summary 202958 (January, 2015).odt</narrative>
        </title>
        <category code="A05" />
        <category code="A04" />
        <category code="A03" />
        <category code="A02" />
        <category code="A01" />
        <language code="en" />
      </document-link>
      <document-link url="http://iati.dfid.gov.uk/iati_documents/4685171.odt" format="application/vnd.oasis.opendocument.text">
        <title>
          <narrative xml:lang="en">Annual review (1) 202958 (May, 2015).odt</narrative>
        </title>
        <category code="A08" />
        <category code="A07" />
        <language code="en" />
      </document-link>
      <document-link url="http://iati.dfid.gov.uk/iati_documents/5062308.odt" format="application/vnd.oasis.opendocument.text">
        <title>
          <narrative xml:lang="en">Annual review (2) 202958 (July, 2015).odt</narrative>
        </title>
        <category code="A08" />
        <category code="A07" />
        <language code="en" />
      </document-link>
      <related-activity ref="GB-1-202958-101" type="2" />
      <related-activity ref="GB-1-202958-102" type="2" />
      <related-activity ref="GB-1-202958-103" type="2" />
      <conditions attached="false" />
    </iati-activity>
    <query>
      <total-count>379</total-count>
      <start></start>
      <limit></limit>
    </query>
  </iati-activities>
</result>
```

**Comment by markbrough** [comment]
This is fantastic work! Even though this was not required at this stage, the data almost perfectly [validates](http://validator.iatistandard.org/index.php?test=xsd) against v2.01 of the IATI Standard - would be perfect with two small changes:

1. Add `version="2.01"` attribute (no namespace) to `<iati-activity>` - this is a valid attribute in v2.x but not v1.x
2. On `<other-identifier>`, text of this element becomes a new attribute, just `ref`. Add `type="A1"` (which means "Reporting Organisation’s internal activity identifier" - this is what this element was mostly used for previously). And then the `owner-name` and `owner-ref` goes into an new element. So what looked like this:

```xml
<other-identifier owner-ref="GB-1" owner-name="DFID">202976-108</other-identifier>
```

Should look like this:

```xml
<other-identifier ref="202976-108" type="A1">
    <owner-org ref="GB-1">
        <narrative>DFID</narrative>
    </owner-org>	  
</other-identifier>
```

**Comment by tvldev02** [comment]
Ok. We have changed accordingly.

However, we have downloaded the v2.01 data from 
http://datastore.iatistandard.org/api/1/access/activity.xml?recipient-country=BD&reporting-org=CA-3&stream=True
here version="2.01" with namespace and other-identifier is also similar to our conversion tool previous result.

You said, version attribute in iati-activity element. In v2.01 schema there is no such a version attribute in iati-activity but in iati-activiteis.

Regards.
